src/index.tsx

```diff
@@ -17,16 +15,16 @@ export function createUniversalPortal(children: any, selector: string) {
   return (ReactDOM as any).createPortal(children, document.querySelector(selector));
 }
 
-export function flushUniversalPortals(): ChildrenSelectorTuple[] {
-  const copy = portals.slice();
-  portals.length = 0;                   // it's important to flush one and only one time per render
-  return copy;
-}
-
-export function removeUniversalPortals() {
+export function prepareClientPortals() {
```

Comment:
From what I see, this method doesn't change its behavior. Only the name has changed to more "ambiguous". It's not obvious what "prepare" means so I would urge to stick to "remove" which is more explicit.

Reply:
I went back and forth on this one internally, but ultimately landed on this rewording.

`prepareClientPortals` implies that "there is some process happening to make these portals work 
on the client". Whereas `removeClientPortals` implies "we're going to remove these portals. And it's permanent." (because there's no equivalent `reAddClientPortals`). I also felt that `remove` is leaking internal implementation details that the user of this library really shouldn't have to care about.